src/components/navbar/navbar.component.tsx

```diff
@@ -93,6 +106,24 @@ const Navbar: React.FC<NavbarProps> = ({
                 >
                   <UserAvatarFilledAlt20 />
                 </HeaderGlobalAction>
+                <HeaderGlobalAction
+                  aria-label="Toggle App Menu"
+                  onClick={handleSwitcherClick}
+                >
+                  <Icon />
+                </HeaderGlobalAction>
+                <HeaderPanel
+                  aria-label="Toggle App Menu Panel"
+                  style={{
+                    display: switcherToggler ? "block" : "none",
+                    height: "12rem",
+                    overFlow: "overflow"
+                  }}
+                  expanded
```

Comment:
Just a question: Wouldn't toggling `expanded` be better than toggle between block and none?

Reply:
Hi @FlorianRappl , I tried this ->  **_expanded = switcherToggler ? 'expanded' : ''_** . But it didn't work. I don't if this is what you want me to do.

Reply:
Hi @FlorianRappl, Is everything okay now. I'm actually excited about this learning process. Is there any other I need to change ?